components/Badge/badge.css

```diff
@@ -0,0 +1,93 @@
+.avatar{
+	border:1px solid rgb(218, 218, 218);
+}
+
+.badge-container{
+	display: inline-flex;
+	position: relative;
+	margin: 0.75rem;
+}
+
+.badge{
+	width: 0.5rem;
+	height: 0.5rem;
+	border-radius: 50%;
+	border: 2px solid white;
+}
+
+.badge-status, .badge-icon{
+	position: absolute;
+}
+
+.badge-container .avatar-lg ~ .badge-status {
+	bottom: 0.7rem;
+	right: 1rem;
+	min-width: 1.5rem;
+	height: 1.5rem;
+}
+
+.badge-container .avatar-md ~ .badge-status {
+	bottom: 0.5rem;
+	right: 0.9rem;
+	min-width: 1.25rem;
+	height: 1.25rem;
+}
+
+.badge-container .avatar-sm ~ .badge-status {
+	bottom: 0.27rem;
+	right: 0.7rem;
+	min-width: 1.1rem;
+	height: 1.1rem;
+}
+
+.badge-container .avatar-xs ~ .badge-status {
+	bottom: 0;
+	right:0.25rem;
+	min-width: 0.8rem;
+	height: 0.8rem;
+}
+
+.badge-status-online{
+	background-color: var(--clr-green);
+}
+
+.badge-status-away{
+	background-color: var(--clr-gray);
+}
+
+.badge-status-dnd{
+	background-color: var(--clr-gray-light);
+}
+
+.badge-status-ill{
+	background-color: var(--clr-red);
+}
+
+.badge-icon{
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	color: white;
```

Comment:
Suggetion: You should use a variable for static color as well, bcuz it may use multiple places in large file.